src/lightning/__setup__.py

```diff
@@ -31,11 +31,15 @@ def _adjust_manifest(**kwargs: Any) -> None:
     with open(manifest_path) as fp:
         lines = [ln.rstrip() for ln in fp.readlines()]
     lines += [
+        "include requirements/base.txt",
         "recursive-include src/lightning *.md",
-        "recursive-include requirements *.txt",
         "recursive-include src/lightning/app/ui *",
         "recursive-include src/lightning/cli/*-template *",  # Add templates as build-in
         # fixme: this is strange, this shall work with setup find package - include
```

Comment:
does this fixme apply to the newly added lines?

Reply:
yes, not sure why these files are included :/